src/problem2.rb

```diff
@@ -0,0 +1,9 @@
+class Problem2
+  def identify_largest_numbers(array, no_elements_identify)
+    unless array.nil?
+      array.max(no_elements_identify)
```

Comment:
Clever, but the problem required you not to sort the array, if you read the [docs](http://ruby-doc.org/core-2.4.1/Enumerable.html#method-i-max) max will sort the array.

Please implement this without sorting.